src/index.js

```diff
@@ -68,11 +48,7 @@ function bindEvents (options) {
     documentCapture: () => options.onDocumentCapture(getCaptures().documentCapture),
     documentBackCapture: () => options.onDocumentCapture(getCaptures().documentBackCapture),
     faceCapture: () => options.onFaceCapture(getCaptures().faceCapture),
-    complete: () => options.onComplete(getCaptures()),
-    onError: () => {
```

Comment:
I think it might still be important to at least track on Sentry 500 exceptions that happen on the JS client. tracking exceptions that happen due to authorization might be worth it too.